generative_models/AAE_PyTorch/README.md

```diff
@@ -0,0 +1,100 @@
+# Adversarial Autoencoder
+## Usage
+```bash
+> python main.py --arguments
+```
+The arguments are as follows-
+```bash
```

Comment:
change \```bash -> \``` , this will fix weird syntax highlighting.

Reply:
Done